pos_reuse_pricelist/static/src/js/models.js

```diff
@@ -0,0 +1,27 @@
+/*
+ *  Copyright 2021 Akretion
+ *  License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+ */
+
+odoo.define("pos_reuse_pricelist.models", function (require) {
+    "use strict";
+
+    var models = require("point_of_sale.models");
```

Comment:
```suggestion
    const models = require("point_of_sale.models");
```

Replace all `var` with `const`

https://www.freecodecamp.org/news/var-let-and-const-whats-the-difference/